src/Core/Content/Flow/Dispatching/FlowDispatcher.php

```diff
@@ -54,7 +56,10 @@ public function dispatch(object $event, ?string $eventName = null): object
         }
 
         if (Feature::isActive('FLOW_EXECUTION_AFTER_BUSINESS_PROCESS')) {
-            $this->container->get(BufferedFlowQueue::class)->queueFlow($event);
+            $this->container->get(MessageBusInterface::class)->dispatch(new FlowMessage(
+                $event,
```

Comment:
Right now this dispatches the event itself right? I think we should better use the existing concept of "storableFlows" as I mentioned in the [ADR](https://github.com/shopware/shopware/pull/13128)

You can see it even being used in the lines below:
https://github.com/shopware/shopware/blob/9e7050dde0075a45c1b61d67eb7e226c26e5f4df/src/Core/Content/Flow/Dispatching/FlowDispatcher.php#L67-L68